ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -122,7 +121,10 @@ def searchsorted(self, v, side="left", sorter=None):
 
     def squeeze(self, axis=None):
         return np_frontend.squeeze(self.data, axis=axis)
-
+    #Add astype instance method
+    def astype(self,dtype,order="K",casting="unsafe",subok=True,copy=True):
```

Comment:
You haven't implemented the `test_function` which should be implemented to complete the task, I'll link you to the document which would help you in the process.
https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html#frontend-instance-method-tests
Also, there are some lint errors which also need to be fixed 🙂